certificate.proto

```diff
@@ -35,6 +47,18 @@ message NewCertificatePayload {
     // for certificate signing. As far as public key will be
     // a part of transaction payload
     string cert_signer_public_key = 4;
+    PubKeyType public_key_type = 5;
+    // For future ACME support(Now only Personal)
+    EntityType entity_owner_type = 6;
+    RSAPayload rsa_parameters = 7;
+    // hash of PEM encoded public key of certificate
+    string entity_hash = 8;
```

Comment:
we also need entity_hash_signature in creation payload. 
@eugene-babichenko  agree with that ?

Reply:
This is `crt_sig`.

Reply:
Rename `crt_sig` -> `entity_hash_signature`
Remove `rem_signature`